docs/capi-controlplane-bootstrap.md

```diff
@@ -63,3 +63,19 @@ spec:
 By applying this yaml, k0smotron will create 3 machines based on the `MachineTemplate` configuration, installs k0s with the role controller on each machine and bootstraps the k0s control plane.
 
 For a full reference on `K0sControlPlane` configurability see the [reference docs](resource-reference.md#controlplaneclusterx-k8siov1beta1).
+
+## Downscaling the control plane
+
+**WARNING: Downscaling is a dangerous operation and should only be done if you know what you are doing.**
+
+Kubernetes using etcd as its backing store. It's crucial to have a quorum of etcd nodes available at all times. Always run etcd as a cluster of **odd** members.
+    
+When downscaling the control plane, you need firstly to deregister the node from the etcd cluster. This can be done by running the following command on the node that will be removed:
+
+```bash
+k0s etcd leave
```

Comment:
Should the user also stop k0s first?

Reply:
Stopping k0s will stop etcd as well and `k0s etcd leave` won't work

Reply:
Ah, of course 🤦